### PR TITLE
docs(add link stackblitz)

### LINKED ISSUE #137: docs: add a link to a stackblitz example
Create a StackBlitz with example usage of our form and validator packages. 

Then update our README with a link to the StackBlitz :)

### PR DESCRIPTION
**Pull request by munyanezaarmel**
## docs(add stackblitz link in Readme.md): description

Fixes #137 

Description of changes: 
- added stackblitz link in readme

Tag a reviewer: @ayoayco 

Tasks:
- [x] I have ran the build command to make sure apps are working: `npm run build`
- [x] I have ran the tests to make sure nothing is broken: `npm run test`
- [x] I have ran the linter to make sure code is clean: `npm run lint`
- [x] I have reviewed my own code to remove changes that are not needed

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,10 @@
     src="https://img.shields.io/badge/Contribute%20with-Gitpod-908a85?logo=gitpod"
     alt="Contribute with Gitpod"
   />
+
+  [![Open in StackBlitz](https://developer.stackblitz.com/img/open_in_stackblitz_small.svg)](https://stackblitz.com/edit/github-ze9ebb-tthuka?file=package.json,src%2Fpages%2Findex.astro,src%2Fcomponents%2FTour.astro)
```
Comment (ayoayco) [inline README.md:21]: Can we say "Play around in Stackblitz" instead?

**Review by ayoayco — CHANGES_REQUESTED** [review]
One suggestion!

File: README.md (hunk 1 above)
Comment (munyanezaarmel) [reply README.md]: @ayoayco done

**Review by ayoayco — APPROVED** [review]
Thanks, great work! 🚀

**Pull request merged**